paying_for_college/templates/worksheet.html

```diff
@@ -1410,7 +1689,7 @@ <h4 class="aid-form_summary-heading">
                                             $
                                         </span>
                                         <span class="line-item_amount"
-                                        data-financial="totalRepayment"
+                                        data-financial="totalDebt"
```

Comment:
All the variable changes are there to reflect the names of variables in `student-debt-calc`.

That said, this is my mistake - this should be a different thing, and I might have to modify student-debt-calc to add this value... :grin: